endpoints/setuid.go

```diff
@@ -125,7 +125,7 @@ func NewSetUIDEndpoint(cfg *config.Configuration, syncersByBidder map[string]use
 				handleBadStatus(w, http.StatusBadRequest, metrics.SetUidBadRequest, err, metricsEngine, &so)
 				return
 			}
-			w.Write([]byte("Warning: " + err.Error()))
+			w.Header().Add("Warning", err.Error())
```

Comment:
We'd rather not add a warning as a header. Instead, I think we should skip sending a warning in the response altogether. We can simply delete this line which should cause the endpoint to set the cookie when both GDPR and GPP are set.

I don't think this is necessarily required but you could explore adding a metric to surface the warning.

Reply:
as per the suggestion, removed the line